Common/Addresses/EquipmentAddresses.cs

```diff
@@ -0,0 +1,188 @@
+using System.Collections.Generic;
+
+namespace CrowdControl.Games.Packs.KH2FM;
+
+public static class EquipmentAddresses {
+    public static uint SoraWeaponSlot = 0x2032E020;
+    public static uint SoraValorWeaponSlot = 0x2032EE24;
+    public static uint SoraMasterWeaponSlot = 0x2032EECC;
+    public static uint SoraFinalWeaponSlot = 0x2032EF04;
+    public static uint DonaldWeaponSlot = 0x2032E134;
+    public static uint GoofyWeaponSlot = 0x2032E248;
+
+
+    public static uint SoraArmorSlotCount = 0x2032E030;
+    public static uint SoraArmorSlot1 = 0x2032E034;
+    public static uint SoraArmorSlot2 = 0x2032E036;
+    public static uint SoraArmorSlot3 = 0x2032E038;
+    public static uint SoraArmorSlot4 = 0x2032E03A;
+    public static uint SoraAccessorySlotCount = 0x2032E031;
+    public static uint SoraAccessorySlot1 = 0x2032E044;
+    public static uint SoraAccessorySlot2 = 0x2032E046;
+    public static uint SoraAccessorySlot3 = 0x2032E048;
+    public static uint SoraAccessorySlot4 = 0x2032E04A;
+    public static uint SoraItemSlotCount = 0x2032E032;
+    public static uint SoraItemSlot1 = 0x2032E054;
+    public static uint SoraItemSlot2 = 0x2032E056;
+    public static uint SoraItemSlot3 = 0x2032E058;
+    public static uint SoraItemSlot4 = 0x2032E05A;
+    public static uint SoraItemSlot5 = 0x2032E05C;
+    public static uint SoraItemSlot6 = 0x2032E05E;
+    public static uint SoraItemSlot7 = 0x2032E060;
+    public static uint SoraItemSlot8 = 0x2032E062;
+    public static uint SoraQuickMenuSlot1 = 0x2032F228;
+    public static uint SoraQuickMenuSlot2 = 0x2032F22A;
+    public static uint SoraQuickMenuSlot3 = 0x2032F22C;
+    public static uint SoraQuickMenuSlot4 = 0x2032F22E;
+    public static uint SoraAbilityStart = 0x2032E074;
+
+    /// <summary>
+    /// Creates a dictionary of all of Sora's inventory slots, each with a quantity of 0.
+    /// You can use this to unequip all of Sora's armor, accessories, keyblades, and items,
+    /// or to store the quantities of that inventory.
+    /// 
+    /// Key: Memory address for the inventory slot
+    /// Value: Equipment, item, or accessory assigned to the slot. Initialized to zero.
+    /// 
+    /// </summary>
+    /// <returns>a dictionary with each inventory address and a value of zero</returns>
+    public static Dictionary<uint, ushort> MakeSoraInventorySlotsDictionary() {
+        return new Dictionary<uint, ushort> {
+            { SoraWeaponSlot, 0 },
+            { SoraValorWeaponSlot, 0 },
+            { SoraMasterWeaponSlot, 0 },
+            { SoraFinalWeaponSlot, 0 },
```

Comment:
I think putting 0x0 into a weapon slot potentially crashes the game - May want to just default to the Kingdom Key Constant Value.